src/app/my-settings/components/my-settings-notifications/my-settings-notifications.component.scss

```diff
@@ -0,0 +1,23 @@
+@import "../../../../bootstrap-variables";
+
+.setting-header {
+  padding-bottom: 1rem;
+}
+
+.checkbox-setting {
+  margin: 0;
+  padding: 0.6rem $spacer/2 0 0;
+}
+
+.checkbox-setting-label {
+  margin: 0;
+  padding: 0.6rem $spacer/2 0 0;
+  padding-left: 1.2rem;
+}
+
+.checkbox-setting input.checkbox-setting-input {
```

Comment:
Ich fänds noch gut, wenn hier das Naming äquivalent wäre, wie in den andere css-Files: `.checkbox input.form-check-input`

Reply:
Hab ich angepasst.